cfpb/cfgov-refresh
# Ask answer streamfield

Moves answer page content into streamfield blocks and makes other updates to the answer page layout.

## Additions

- Migration to move answer content into streamfield
- Blocks and templates for tips and answer content

## Removals
- Remove separate `search-bar.html` organism since it was only being used for Ask

## Changes
- Update answer page layout
- Move content into streamfield blocks

## Testing

1. Check out branch
2. Run migration & `gulp styles`
3. Navigate to an English and Spanish answer page. Ensure its content still displays correctly, the search bar works, and the style/layout matches screenshot.
4. Add a tip to the content of an answer page and verify that it displays as in design.

## Screenshots

![answer-page](https://user-images.githubusercontent.com/778171/57245176-0992e700-6fef-11e9-8444-4025676a5dea.png)

## Design
![22fa5280-45ad-11e9-8010-e61f48f7d896](https://user-images.githubusercontent.com/778171/57245329-71493200-6fef-11e9-8c27-07f69a3e1148.png)



## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
